src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -259,6 +259,11 @@ public X509Certificate2Collection ClientCertificates
         {
             get
             {
+                if (_clientCertificateOption != ClientCertificateOption.Manual)
```

Comment:
I'm not sure we should not be changing WinHttpHandler at all.  The bug was about HttpClientHandler only.